pkg/registry/pod/rest.go

```diff
@@ -177,3 +186,42 @@ func ResourceLocation(getter ResourceGetter, ctx api.Context, id string) (*url.U
 	}
 	return loc, nil, nil
 }
+
+// LogLocation returns a the log URL for a pod container. If container name is blank, the first container is used
```

Comment:
Comment needs to be updated similarly to API descriptions: defaults to only container if there is one container in the pod